src/main/c/Objects/pyjarray.c

```diff
@@ -1626,69 +1624,29 @@ PyMethodDef pyjarray_methods[] = {
     { NULL, NULL }
 };
 
-
-static PySequenceMethods list_as_sequence = {
-    (lenfunc) pyjarray_length,                /* sq_length */
-    (binaryfunc) 0,                           /* sq_concat */
-    (ssizeargfunc) 0,                         /* sq_repeat */
-    (ssizeargfunc) pyjarray_item,             /* sq_item */
-    (ssizessizeargfunc) pyjarray_slice,       /* sq_slice */
-    (ssizeobjargproc) pyjarray_setitem,       /* sq_ass_item */
-    (ssizessizeobjargproc) 0,                 /* sq_ass_slice */
-    (objobjproc) pyjarray_contains,           /* sq_contains */
-    (binaryfunc) 0,                           /* sq_inplace_concat */
-    (ssizeargfunc) 0,                         /* sq_inplace_repeat */
-};
-
-static PyMappingMethods pyjarray_map_methods = {
-    (lenfunc) pyjarray_length,                /* mp_length */
-    (binaryfunc) pyjarray_subscript,          /* mp_subscript */
-    0,                                        /* mp_ass_subscript */
-};
-
-
 static PyObject* pyjarray_iter(PyObject *);
 
+static PyType_Slot slots[] = {
+    {Py_tp_doc, (void*) &list_doc},
```

Comment:
I'm not sure I agree with using list_doc, though I'm not entirely sure what this is used for.  If you open the jep interpreter and make a pyjarray and do .__doc__ on it, what does it say?

Reply:
This is the only place list_doc is used and I am pretty sure it is the same behavior as before. pyjarray.doc is not a valid attribute but __doc__ is a string with list_doc in it. The main purpose of this is so that at an interactive interpreter help(jep.jarray) shows a nice help message.